src/abbfreeathome/devices/switch_sensor.py

```diff
@@ -57,4 +59,16 @@ def _refresh_state_from_output(self, output: dict[str, Any]) -> bool:
         if output.get("pairingID") == Pairing.AL_SWITCH_ON_OFF.value:
             self._state = output.get("value") == "1"
             return True
+        if output.get("pairingID") == Pairing.AL_RELATIVE_SET_VALUE_CONTROL.value:
+            self._longpress = output.get("value") == "9" or output.get("value") == "8"
+            return True
```

Comment:
Would it be useful to have a long press up and a long press down?

If this is the possible values.

- longpress down: 1
- longpress down release: 0
- longpress up: 9
- longpress up release: 8

You could have a state with an enum with the 4 possible options. In Home Assistant they'd appear as one of the 4 options. That way a user can create automations based on one of the 4 options.

Reply:
Let me try it